UE/Tests/Application/SmsDbTestSuite.cpp

```diff
@@ -0,0 +1,36 @@
+#include <gtest/gtest.h>
+#include "Messages/PhoneNumber.hpp"
+#include "Sms.hpp"
+#include "SmsDb.hpp"
+
+namespace ue
+{
+using namespace ::testing;
+
+class SmsDbTestSuite: public Test
+{
+public:
+    static constexpr common::PhoneNumber number{220};
+
+    SmsDb objTested;
+};
+
+TEST_F(SmsDbTestSuite, shallAddUnreadSms)
+{
+    const Sms sms{number, "test suite"};
+    objTested.addOne(sms);
```

Comment:
I am adding this comment here, because here it stands out. If the variable name was different, I wouldn't know what `objTested.addOne` was doing. Maybe a more descriptive name would be useful, but I would like to hear what others think.